modules/rosa-upgrade-cluster-cli.adoc

```diff
@@ -0,0 +1,105 @@
+:_module-type: REFERENCE
+// Module included in the following assemblies:
+//
+// * cli_reference/rosa_cli/rosa-manage-objects-cli.adoc
+
+[id="rosa-upgrade-cluster_{context}"]
+= Upgrade and delete upgrade for clusters
+
+[role="_abstract"]
+This section describes the `upgrade` command usage for clusters.
+
+== upgrade cluster
+
+Schedules a cluster upgrade.
```

Comment:
s/Schedules/Schedule
When describing a task to follow, it's better to make it actionable rather than descriptive, if that makes sense. :)

Reply:
Done.